tests/frameworks/test_grpcio.py

```diff
@@ -690,6 +691,7 @@ def test_root_exit_span(self) -> None:
         assert not server_span.data["rpc"]["error"]
 
     def test_no_root_exit_span(self) -> None:
+        agent.options.allow_exit_as_root = True
```

Comment:
Please remove this line or change it to False. We already have a testcase to check for root exit span when set to True. In this testcase, we check if no spans are recorded when set to False.

```suggestion
        agent.options.allow_exit_as_root = False
```

Reply:
Curiously, the missing of this setting was causing one flaky test error.
I changed the code.